### DISCUSSION TITLE
Grapevine 5: Setting Host and Port of Server

### DISCUSSION THREAD

**Discussion by Kieran-Arul** [scottoffen/grapevine#107] [Q&A] [answered]
Hello, thank you very much for making this library! I really appreciate it.

May I ask, in Grapevine 5, how do I set a host and port other than localhost and port 1234? 

In grapevine 4, it seems that this could be done by creating a RestServer without any parameters then setting the host and port by setting the fields (like server.Host and server.Port). However, that no longer seems possible.

How would I make my server listen on localhost:8080 instead for example?

SOLVED:

Just need to access the Prefixes field of the RestServer object and add another address there.

**Comment by scottoffen** [comment, marked as answer]
You can do this by adding the the `RestServer.Prefixes` collection.
